### ISSUE TITLE
The requirements for start and end states of demonstration sequence?

### ISSUE THREAD

**Issue by xjwxjw** [xbpeng/DeepMimic#90]
Hello Jason,
I would like to ask is there any specific requirements for the start or end states of demonstration sequence?
I am trying to generate motion files based on my own data and I notice that the majority of your examples possess almost the same start and end states.
It seems like cyclic if viewed in "wrap" mode.
Is this a basic condition for training?
Are there any other conditions I need to fulfill?
Thank you very much!

**Comment by Zju-George** [comment]
Of course not. It need not to be cyclic.
When you use your own data, how's your training result ？Could you please show some gifs?

**Comment by xjwxjw** [comment]
Currently not so well, that's why I ask this question.
Suppose the agent reaches end state, what can we expect the following action?
After all, the demonstration itself is not cyclic.

**Comment by Zju-George** [comment]
Sry I'm not clear enough.
If it is **wrap** mode, it need to be cyclic due to the definition, and the agent will like start from the begining but it is just like, you know, it is a whole thing but like it is start over. However, if it is **none** mode, it will end when reaches end state, it is real end.

**Comment by xjwxjw** [comment]
So "wrap" and "none" modes are tested differently?
For "warp" mode it will be tested as long as you want, but "none" mode ends once it finished, right?

**Comment by Zju-George** [comment]
Yeah, exactly. If we don't consider early termination, it is just like you said. In practice, when the agent is hard to recover the right pose, for example, it falls to the ground, the peroid will end immediatly.

**Comment by xjwxjw** [comment]
Get it. Thx.

**Comment by xjwxjw** [comment]
One more question:
Does the framerate greatly affect model training?
The fps used in example file is at least 0.016, but mine is 0.008.
